Changelog.md

```diff
@@ -2,4 +2,13 @@
 
 ## 0.0.1
 
-- Completion to create a Database view in *.ddl files
+- added support for ConfigMap and Secret backed environment variables
+- added ability to create / modify / delete environment variables
+- added support for ConfigMap and Secret backed datasource properties
+- completion to create a Database view in *.ddl files
+- added ability to create / modify / delete data source properties
+- added ability to create / delete data sources
+- added ability to modify the DDL part of the VDB in an editor
+- added extension specific output channel for logging
+- added Data Virtualization tree view to visualize VDB content
+- ability to create a VDB definition file with basic DDL
```

Comment:
I think it would make the changelog more readable if we are grouping entries by type.
For instance all "Configmap/Secrets" and "create/modify/delete"

Reply:
Please raise an issue for it as this PR is already closed.

Reply:
created https://issues.redhat.com/browse/FUSETOOLS2-383